linux/cpu_hot_add_basic/cpu_hot_add_basic.yml

```diff
@@ -15,40 +15,57 @@
   tasks:
     - name: "Test case block"
       block:
-        - include_tasks: ../../common/skip_test_case.yml
+        # Except for VM guest_id like 'fedora64Guest', 'freebsd.*Guest' not supporting vCPU hot-add,
+        # the other guest OS compatible with ansible-vsphere-gos-validation support vCPU hot-add
+        - name: "Skip test case '{{ ansible_play_name }}' due to VM not support vCPU hot-add"
```

Comment:
"VM" --> "guest OS" is better?

Reply:
No. It's because the VM's guest id doesn't support cpu hot-add, no matter what the real guest OS is.